pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -67,7 +67,7 @@ type BindMechanism interface {
 	// The following entry points require domain initialized for the
 	// binding and can be used in phase2 only.
 	decorateConfig() error
-	startDHCP(vmi *v1.VirtualMachineInstance) error
+	startDynamicIPServers(vmi *v1.VirtualMachineInstance) error
```

Comment:
>Since that method is now used to also create the RA server in IPv6,

Since the commit that actually add tge RA creation to the method is just in the following commit, I would update the commit message to - `Since that method will be used to also create the RA server in IPv6,`